.semaphore/semaphore.yml

```diff
@@ -12,9 +12,12 @@ blocks:
       - name: Run Manual tests
         commands:
           - checkout
-          - sem-service start mongodb 
-          - sem-service status mongodb 
-          - sem-service stop mongodb 
-          - sem-version php 7.0.32
-          - sem-version ruby 2.5.3
-          - sem-version ruby 2.5.9
+          - . sem-version
```

Comment:
What's the point of this line?

Reply:
Just to use the latest version of `sem-version` in the VM. It will be removed once the changes are accepted.